metadata-io/src/test/java/com/linkedin/metadata/search/elasticsearch/fixtures/SampleDataFixtureTests.java

```diff
@@ -1126,12 +1126,13 @@ public void testPrefixVsExact() {
         assertTrue(result.getEntities().stream().noneMatch(e -> e.getMatchedFields().isEmpty()),
                 String.format("%s - Expected search results to include matched fields", query));
 
-        assertEquals(result.getEntities().size(), 10);
+        assertTrue(result.getEntities().size() > 1);
```

Comment:
This seems like a concerning change happened? Why is the size not the same anymore?

Reply:
Will look into this closer before continuing.

Reply:
Ok, this line can be reverted to the original. I was experimenting with the customization at one point. I will restore the original assertion and it still works. Thanks!